### PR TITLE
@alloy -> fix broken explore layout won rotation when another tab is selected

### PR DESCRIPTION
**Pull request by 1aurabrown**
fixes https://github.com/artsy/eigen/issues/395

### CODE REVIEW

File: Artsy/Classes/View Controllers/ARBrowseViewController.m [hunk 1]
```diff
@@ -123,7 +129,7 @@ - (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollection
     if ([UIDevice isPhone]) {
         heightToWidthFactor = .597;
     } else {
-        if (UIInterfaceOrientationIsLandscape(self.interfaceOrientation)) {
+        if (self.view.frame.size.width > self.view.frame.size.height) {
```
Comment (alloy) [inline Artsy/Classes/View Controllers/ARBrowseViewController.m]: Btw, there are `CGRectGetWidth` and `CGRectGetHeight` functions, they might be uuuuuuber faster!

**Comment by alloy** [comment]
Can you add a beta changelog? :)

**Comment by 1aurabrown** [comment]
I forgot about those guys. Will update

On Fri, Apr 24, 2015 at 14:09, artsy/eigen
<reply@reply.github.com>
wrote:
Can you add a beta changelog? :)

—
Reply to this email directly or view it on GitHub
[https://github.com/artsy/eigen/pull/400#issuecomment-96017817] .[https://github.com/notifications/beacon/ADBlTpU6FAhGCkfj8vQDHdROpov2_6ukks5oCn5JgaJpZM4EH_wB.gif]

**Comment by 1aurabrown** [comment]
@alloy updated

**Comment by alloy** [comment]
👌

**Pull request merged**